src/components/Input.tsx

```diff
@@ -177,14 +179,18 @@ const Input: React.FC = () => {
                         : `${displayFormat}${asSingle ? "" : ` ${separator} ${displayFormat}`}`
                 }
                 value={inputText}
+                id={inputId}
+                name={inputName}
+                autoComplete="off"
+                role="presentation"
```

Comment:
These are optional, but should insure other password managers also leave these fields alone.